README.md

```diff
@@ -1,22 +1,21 @@
-# Language Model Evaluation Harness
+# Promptsource X Language Model Evaluation Harness
 
 ![](https://github.com/EleutherAI/lm-evaluation-harness/workflows/Build/badge.svg)
 [![codecov](https://codecov.io/gh/EleutherAI/lm-evaluation-harness/branch/master/graph/badge.svg?token=JSG3O2427J)](https://codecov.io/gh/EleutherAI/lm-evaluation-harness)
 
 ## Overview 
 
-This project provides a unified framework to test autoregressive language models (GPT-2, GPT-3, GPTNeo, etc) on a large number of different evaluation tasks.
+This project provides a unified framework to test language models (GPT-2, GPT-3, GPTNeo, etc) and seq2seq (T5, T0) models via prompt evaluation. The prompts are provided via `promptsource`; all datasets are in huggingface `datasets`.
```

Comment:
This is not true. The Hackathon is using datasets in HF, but the eval harness does not require the dataset to be in HF. My eventual goal is to merge this fork into the EleutherAI version, so comments like this should be avoided where possible.

Reply:
I did not know that was in the works. Currently, if the dataset either is not in HF or not in promptsource (unless specifically handled), the code won't run.

Reply:
I added some more clarification -- I didn't want someone to try to merge it back or just use the fork into their fork or something. There'd be lots of issues at the moment.

Reply:
All datasets must be wrapped by a HuggingFace `datasets` loading script. If a user wants to run on a  custom dataset they need to write such HF loading scripts as we do in our `lm_eval/datasets`. This is the case in the `master` branch and the big science one. (If you want to know more about this design decision, I can copy and paste the discussion I had with Leo on discord about it here.)

Reply:
@jon-tow Oh interesting, I hadn’t realized that. To be clear, the datasets directory says
> This directory contains custom EleutherAI datasets not available in the HuggingFace datasets hub.
and this _is correct_, however these custom datasets are loaded using the library? They’re just local rather than out on the web?

if that’s correct we should do another pass at the language here be user I don’t think that people will reliably reach that conclusion based on the current wording.

Reply:
_Re the design decision_: 
- Long-story-short; we had 2 competing dataset downloading systems: our custom ad-hoc downloaders (usually inconsistent with each other) and the HuggingFace `datasets` download manager. We opted to go all-in on HF `datasets` because it was significantly more robust in caching and error handling.

> and this is correct, however, these custom datasets are loaded using the library? They’re just local rather than out on the web?

- Yes. These custom datasets are downloaded using the `datasets` library by loading from the local loading scripts; you can point local file paths to `datasets.load_dataset` instead of pointing to HF [Hub](https://huggingface.co/datasets) or a [repo](https://huggingface.co/docs/datasets/repository_structure). I initially requested write access to [EleutherAI's](https://huggingface.co/EleutherAI) HuggingFace Organization to place these custom datasets there but I never got a response nor bumped for a follow-up. Instead, I opted to dump them locally in the harness. 

> if that’s correct we should do another pass at the language here 
- I'll reword that `README.md` tomorrow!